types/yeoman-generator/index.d.ts

```diff
@@ -227,6 +227,21 @@ declare namespace Generator {
         path: string;
     }
 
+    /**
+     * Represents generators feature
+     */
+    interface GeneratorFeatures {
```

Comment:
You might want to add all the other features (`uniqueBy`, `unique`, `tasksMatchingPriority`) the user could provide:
https://github.com/yeoman/generator/blob/c1c847dbed005e40ded29ea46d1efd4dbac5cc51/lib/index.js#L122-L130